conf/polarion_tools.yaml

```diff
@@ -131,6 +131,7 @@ docstrings:
             - Provisioning
             - Quota
             - Replication
+            - HAProxy
```

Comment:
I have added this casecomponent to Polarion, please don't change this string.

Reply:
OK